libs/components/lookup/src/lib/modules/lookup/lookup-autocomplete-adapter.ts

```diff
@@ -30,25 +30,33 @@ export class SkyLookupAutocompleteAdapter {
    * @default "name"
    */
   @Input()
-  public set descriptorProperty(value: string) {
-    this.#_descriptorProperty = value;
+  public set descriptorProperty(value: string | undefined) {
+    this.#_descriptorProperty = value ?? 'name';
   }
 
   public get descriptorProperty(): string {
-    return this.#_descriptorProperty || 'name';
+    return this.#_descriptorProperty;
   }
 
   /**
    * Specifies an array of object properties to search.
    * @default ["name"]
    */
   @Input()
-  public set propertiesToSearch(value: string[]) {
-    this.#_propertiesToSearch = value;
+  public set propertiesToSearch(value: string[] | undefined) {
+    this.#_propertiesToSearch = value ?? ['name'];
+
+    // Reset default search if it is what is being used.
+    if (this.search !== this.searchOrDefault) {
+      this.searchOrDefault = skyAutocompleteDefaultSearchFunction({
+        propertiesToSearch: this.propertiesToSearch,
+        searchFilters: this.searchFilters,
+      });
+    }
   }
 
   public get propertiesToSearch(): string[] {
-    return this.#_propertiesToSearch || ['name'];
```

Comment:
This was a particularly nasty violation of the "don't return default values in a getter" because it would return a new array every time 🤦